### ISSUE TITLE
Search by genus: internal server error

### ISSUE THREAD

**Issue by plantarum** [ropensci/natserv#27]
<details> <summary><strong>Reproducible example</strong></summary>
Searching by Kingdom works:

```r
ns_search_spp(species_taxonomy = list(scientificTaxonomy = "Animalia", level = "kingdom"))
$results
# A tibble: 20 × 15
   recordType elementGlobalId uniqueId    nsxUrl         elcode scientificName  
   <chr>                <int> <chr>       <chr>          <chr>  <chr>           
 1 SPECIES             828458 ELEMENT_GL… /Taxon/ELEMEN… AAABC… Acris blanchardi
 2

....
```

Searching by genus fails:

```r
> ns_search_spp(species_taxonomy = list(scientificTaxonomy = "Acris", level = "genus"))
Error: Internal Server Error (HTTP 500)
```

</details>

<details> <summary><strong>Session Info</strong></summary>

```r
> sessionInfo()
R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_CA.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_CA.UTF-8        LC_COLLATE=en_CA.UTF-8    
 [5] LC_MONETARY=en_CA.UTF-8    LC_MESSAGES=en_CA.UTF-8   
 [7] LC_PAPER=en_CA.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_CA.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] natserv_1.0.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8      fansi_1.0.2     utf8_1.2.2      crayon_1.4.2   
 [5] crul_1.2.0      R6_2.5.1        lifecycle_1.0.1 jsonlite_1.7.3 
 [9] magrittr_2.0.1  pillar_1.6.4    cli_3.1.0       rlang_0.4.12   
[13] curl_4.3.2      vctrs_0.3.8     ellipsis_0.3.2  urltools_1.7.3 
[17] tools_4.1.2     triebeard_0.3.0 compiler_4.1.2  pkgconfig_2.0.3
[21] tibble_3.1.6    httpcode_0.3.0 
```
</details>

**Comment by plantarum** [comment]
This is a duplicate of #25, and appears to be fixed by #26

**Comment by mikeroswell** [comment]
Hi @plantarum , maybe you saw this but the fix in #26 is a pull request so right now, the package from ropensci won't have the fix but the forked version from me should. Rumor has it this package is in search of a maintainer, so if you're interested, I think the gig is yours!

**Comment by plantarum** [comment]
Thanks @mikeroswell. I saw your issue & pull request, and did some git voodoo to slurp the changes from your fork into mine.

I'm interested in contributing, but not sure I have enough time to take on formal responsibility. Maybe I can make the time if I could get a publication out of it? For now I just need to figure out enough to get this working for my immediate needs.
